code/modules/reagents/newchem/drugs.dm

```diff
@@ -551,7 +551,7 @@
 /datum/reagent/lube/ultra/addiction_act_stage4(var/mob/living/carbon/human/M as mob)
 	M.Jitter(20)
 	M.Dizzy(20)
-	M.adjustToxLoss(5)
+	M.adjustBrainLoss(2)
```

Comment:
Very nice. Thank you for taking care of this! If you want, you can also add NO_POISON to the IPC flags. And yes, increasing the health of the mmi_holder would be wise.